decisionengine/decisionengine_endpoint/recieve_rock_info.py

```diff
@@ -40,13 +40,32 @@ def find_path():
         if request.get_header(head) != headers.get(head):
             abort(406, 'Incorrect headers')
     rocks = request.json.get('rocks')
-    print(rocks)
     if not rocks:
         abort(400, 'Bad request')
     for rock in rocks:
         if not (rock.get('placement') or rock.get('type')):
             abort(400, 'Bad request')
 
-    return {'Message': 'Request successfully recieved, congratz on your rockz', 'Response rocks': rocks}
+    userHeightRequest = request.json.get('userHeight')
+    if userHeightRequest is None:
+        userHeightRequest = '' 
+    elif not isinstance(userHeightRequest, (int, float)):
+        abort(400, 'Bad user height number input')
 
+    wallHeightRequest = request.json.get('wallHeight')
+    if wallHeightRequest is None:
+        wallHeightRequest = '' 
+    elif not isinstance(wallHeightRequest, (int, float)):
+        abort(400, 'Bad wall height number input')
+
+    endPts = {}
```

Comment:
Can we rename endPts to something that makes maybe like params?

Reply:
fixed